src/Application/Console/Formatters/FormatResolver.php

```diff
@@ -29,22 +28,28 @@ public static function resolve(
         OutputInterface $output,
         OutputInterface $consoleOutput
     ): Formatter {
-        $requestedFormat = $input->getOption('format');
+        $requestedFormats = $input->getOption('format');
 
-        if (! is_string($requestedFormat)) {
-            throw new InvalidArgumentException(
-                'Format has to be a string.'
-            );
+        if (! is_array($requestedFormats)) {
+            $consoleOutput->writeln('<fg=red>Could not understand requested format, using fallback [console] instead.</>');
+            $requestedFormats = ['console'];
         }
 
-        $requestedFormat = strtolower($requestedFormat);
+        $formatters = [];
+        foreach ($requestedFormats as $requestedFormat) {
+            if (class_exists($requestedFormat)) {
+                $formatter = $requestedFormat;
```

Comment:
```suggestion
                $formatters[] = new $requestedFormat($input, $output);
                continue;
```

To avoid a else with an if inside